cmd/flags/cassandra/options.go

```diff
@@ -34,6 +34,12 @@ const (
 	suffixSocketKeepAlive  = ".socket-keep-alive"
 	suffixUsername         = ".username"
 	suffixPassword         = ".password"
+	suffixTLS              = ".tls"
+	suffixCert             = ".cert"
```

Comment:
I think it would better to prefix all params with `tls-`, e.g. `.tls-cert`, `.tls-key`, etc.